packages/web/e2e/socials.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from '@playwright/test'
+import { SocialActions } from './page-object-models/socialActions'
+import { test } from './test'
+
+test('should favorite/unfavorite a track', async ({ page }) => {
+  await page.goto('sebastian12/bachgavotte-1')
+
+  const socialActions = new SocialActions(page)
+  await expect(
+    socialActions.favoriteButton.or(socialActions.unfavoriteButton)
```

Comment:
love the use of `.or` here

Reply:
Oh yeah this is clean! Nicer than what I did to achieve this